### PR TITLE
Add MeterReportService collectBatch method.

### PR DESCRIPTION
**Pull request by liqiangz**
Add MeterReportService collectBatch method.

apache/skywalking#6210

### CODE REVIEW

**Comment by liqiangz** [comment]
@wu-sheng Could you please have review on this?

File: language-agent/MeterCompat.proto [hunk 1]
```diff
@@ -30,4 +30,8 @@ service MeterReportService {
     // Meter data is reported in a certain period. The agent/SDK should report all collected metrics in this period through one stream.
     rpc collect (stream skywalking.v3.MeterData) returns (skywalking.v3.Commands) {
     }
+
+    // It is recommended to keep the stream alive for performance considerations
+    rpc collectBatch (stream skywalking.v3.MeterDataCollection) returns (skywalking.v3.Commands) {
+    }
```
Comment (wu-sheng) [inline language-agent/MeterCompat.proto:36]: We don't need this. Compat is for old release, clearly, there would not be any old release implemented this new service.

File: language-agent/Meter.proto [hunk 2]
```diff
@@ -30,6 +30,10 @@ service MeterReportService {
     // Meter data is reported in a certain period. The agent/SDK should report all collected metrics in this period through one stream.
     rpc collect (stream MeterData) returns (Commands) {
     }
+
+    // It is recommended to keep the stream alive for performance considerations
```
Comment (wu-sheng) [inline language-agent/Meter.proto:34]: I think this is not a good command for a service.

```suggestion
    // Reporting meter data in bulk mode as MeterDataCollection.
    // By using this, each one in the stream would be treated as a complete input for MAL engine.
    // Comparing to `collect (stream MeterData)`, one stream is an input data set.
```

Comment (liqiangz) [reply language-agent/Meter.proto]: Updated.

File: language-agent/MeterCompat.proto (hunk 1 above)
Comment (liqiangz) [reply language-agent/MeterCompat.proto]: Removed.

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
Please continue. FYI @gxthrj @kezhenxu94 @mrproliu @wankai123 We have meter report service update.

**Pull request merged**